src/store/thunk/getPeople.ts

```diff
@@ -0,0 +1,43 @@
+import { AnyAction } from 'redux'
+import { ThunkAction, ThunkDispatch } from 'redux-thunk';
+
+import { getPeopleSuccess, getPeopleFail, getPeopleA } from '../actions';
+import { PeopleCard } from '../../global-models';
+import { Image, RequestPeople } from './models/get-people-types';
+
+const requestPeople = () => fetch('https://swapi.co/api/people/');
+const requestRandomImages = () => fetch(`https://api.unsplash.com/photos/?client_id=${process.env.ACCESS_KEY}`);
```

Comment:
What if we'll do a separate layer for the API calls?  lets create API folder for handling API calls and just import them where it's needed